profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.module

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @file
+ * Implements some specific logic for platform feature sets.
+ */
+
+/**
+ * Implements hook_FORM_ID_alter().
+ */
+function nexteuropa_feature_set_form_feature_set_admin_form_alter(&$form, &$form_state) {
+  $featuresets = feature_set_get_featuresets();
+  if (!empty($featuresets)) {
+    foreach ($featuresets as $featureset => $details) {
+      if (!user_access('manage feature ' . $featureset)) {
```

Comment:
The string 'manage feature ' should be defined in a variable to avoid future syntax errors.